src/routes/runtime-stats.tsx

```diff
@@ -1,10 +1,266 @@
+//Sources: https://www.youtube.com/watch?v=InMnGwP7iX4
+//         https://stackoverflow.com/questions/24502898/show-or-hide-element-in-react?rq=3
 import { GroupFooter, NavBar } from "../components/general";
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import {
+  Chart as ChartJS,
+  CategoryScale,
+  LinearScale,
+  BarElement,
+  PointElement,
+  Title,
+  Tooltip,
+  Legend,
+  TimeScale,
+} from 'chart.js/auto';
+import { Scatter, Bar } from 'react-chartjs-2';
+import 'chartjs-adapter-date-fns';
+
+
+ChartJS.register(
+  CategoryScale,
+  LinearScale,
+  BarElement,
+  PointElement,
+  Title,
+  Tooltip,
+  Legend,
+  TimeScale
+);
+
+export const options = {
```

Comment:
unless you plan on moving this to another file, you don't need to have the export here. I suppose it could be useful for other who may want to use the same options, but I think it's safe to just make `const` and not export it.